edward/inferences.py

```diff
@@ -199,12 +199,10 @@ def build_score_loss(self):
         ELBO = E_{q(z; lambda)} [ log p(x, z) - log q(z; lambda) ]
         """
         x = self.data.sample(self.n_data)
-        z, self.samples = self.variational.sample(self.n_minibatch)
-
-        q_log_prob = tf.zeros([self.n_minibatch], dtype=tf.float32)
-        for i in range(self.variational.num_factors):
-            q_log_prob += self.variational.log_prob_i(i, tf.stop_gradient(z))
+        self.samples = self.variational.sample(self.n_minibatch)
+        z = self.samples
```

Comment:
I'll use `self.zs` because we use `zs` in `log_prob(xs, zs)` methods in the model. This makes it more consistent. We can think about whether it should be more verbose that it represents samples explicitly in the variable name or in the docs later on.